SECURITY.md

```diff
@@ -0,0 +1,69 @@
+# Security Policy
+
+## Supported Versions
+
+We recommend to only use the most recent minor release.
```

Comment:
This is where a repo specific security.md file would make sense. So node for example can list their specific support versions which will be different from core, app, etc.

Reply:
Then perhaps we should not define what release versions are supported in this policy, but let each repo define it, and just say in case it's not defined, we recommend the most recent minor release?

Reply:
Yea makes, sense. I'll re-word to make that more clear.